.github/workflows/ci.yaml

```diff
@@ -0,0 +1,13 @@
+name: ci
+
+on: [push, pull_request]
```

Comment:
This means that it will run when a new PR occurs and also when it's pushed?

Reply:
It should trigger the check when a new PR is created or there's a new commit in a branch

Reply:
Ahh I see ...right make sense.Thank you